### PR TITLE
Update summary.md

### PR DESCRIPTION
**Pull request by cefoo**

### CODE REVIEW

File: applications/advanced-concepts/arbitrage.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+description: Arbitrage for machine translation
```
Comment (bittlingmayer) [inline applications/advanced-concepts/arbitrage.md:2]: "arbitrage" or "engine picking" is how I think of it, but I'm not sure what the most common name is for this approach.

https://en.wikipedia.org/wiki/Hybrid_machine_translation

"hybrid" to me implies a mix of human and machine, whereas this is a mix of multiple machines.

I might call it "multi-engine machine translation".

File: business/products.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+description: Providers of machine translation services
+---
+
+# Providers
+
+Machine translation **providers** offer generic machine translation services with different customization possibilities. Machine translation services have different features and limitations.
```
Comment (bittlingmayer) [inline business/products.md:7]: This is kind of hollow.  "different", "various", "several" is often a sign of that.  Anyway, this article isn't really about how the providers offer customisation or differ, so that could go further below, but not in the first sentences.

> Machine translation **providers** offer ready-to-use machine translation systems.  Typically the systems are provided as APIs, and also as applications or integrations.

> They vary by:
> - language support
> - accuracy
> - price
> - customization options
> - data confidentiality
>
> and other features, as well as underlying architecture.

File: business/products.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+The advantage of using a machine translation provider is that the service is already working and can be implemented immediately. The alternative is to train a machine translation engine from scratch.
```
Comment (bittlingmayer) [inline business/products.md:9]: "integrated", not "implemented"

"The alternative is to train a model from scratch or to use a pretrained model, and deploy and maintain a machine translation system."

File: customization/parallel-data.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+description: Parallel data for machine translation
+---
+
+# Parallel data
+
+**Parallel data** is a collection of sentences and their corresponding translation. Parallel data is aligned line-by-line. In machine translation, parallel data is also called parallel corpora.
```
Comment (bittlingmayer) [inline customization/parallel-data.md:7]: **Parallel data** or **parallel corpora** are datasets of **translation pairs** - sentences and their translations.  They are used to train and test machine translation models.

| Original         | Translation  |
| -------------|-------------|
| `Hello`         |    `Hola`          |

Parallel datasets can include translations for one or more language pairs, and be directioned or directionless.

Parallel data can be created by crawling and aligned monolingual test, and by [back-translation] or [back-copying].

File: applications/advanced-concepts/arbitrage.md (hunk 1 above)
Comment (cefoo) [reply applications/advanced-concepts/arbitrage.md:2]: I think "hybrid" is the best way to describe the half-machine, half-human workflow.
"Artitrage" is probably a difficult term that misses the scope of the site. "Engine picking" seems to be the most transparent term to describe this topic.
To me, "multi-engine machine translation" takes the focus out of the process of evaluating and choosing the best fit, it just concentrates of the results.

Comment (bittlingmayer) [reply applications/advanced-concepts/arbitrage.md:2]: It's the most transparent, but I'm leery of inventing a term.

I did some searching now, as far as I can tell, "multi-engine machine translation" has the most currency in the literature.

File: applications/advanced-concepts/multi-engine-machine-translation.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+---
+description: Multi-engine machine translation
+---
+
+# Multi-engine machine translation
+
+**Multi-engine machine translation** is the process of evaluating and picking a machine translation engine. In multi-engine machine translation, different [providers](business/providers.md) translate the same input sample. The output versions are then evaluated automatically or by a human to select a provider.
```
Comment (bittlingmayer) [inline applications/advanced-concepts/multi-engine-machine-translation.md:7]: Is it a "process"?  I'd probably say an "approach".

Comment (bittlingmayer) [reply applications/advanced-concepts/multi-engine-machine-translation.md:7]: > With multi-engine machine translation, the same input is sent to multiple machine-translation systems.

(Systems is neutral - it includes both models and providers and other things)

> The translations are then ranked to select the best one.

File: applications/advanced-concepts/multi-engine-machine-translation.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+Multi-engine machine translation is challenging because it is subjective.
```
Comment (bittlingmayer) [inline applications/advanced-concepts/multi-engine-machine-translation.md:9]: I'd say the challenges are:
- consistency across segments
- complexity, latency and cost

i.e. accuracy challenges, vs practical implementation challenges

File: applications/advanced-concepts/multi-engine-machine-translation.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+### Goals
+
+The goal of multi-engine machine translation is to find the best fit for specific machine translation needs.
```
Comment (bittlingmayer) [inline applications/advanced-concepts/multi-engine-machine-translation.md:13]: > The goal of multi-engine machine translation is to get higher quality translations than any single engine could achieve.

File: applications/advanced-concepts/multi-engine-machine-translation.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+The best fit for a particular input will vary depending on different factors:
+
+- Domains
```
Comment (bittlingmayer) [inline applications/advanced-concepts/multi-engine-machine-translation.md:16]: I think we should remove all of this because these are more project-level criteria, but the engine selection can be segment-level, and the benefit is really avoiding random catastrophic junk (i.e. limiting downside, not increasing quality in the ideal cases).

File: business/data-confidentiality.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
+---
+description: Data confidentiality in machine translation
+---
+
+# Data confidentiality
+
+**Data confidentiality** refers to how the machine translation providers use the specific data sent for translation. Providers can generate translations with or without storing the input. Machine translation providers need data to train and improve their translation service.
```
Comment (bittlingmayer) [inline business/data-confidentiality.md:7]: > **Data confidentiality** terms or policies govern how machine translation providers use client data.

> Machine translation providers use client data to train and improve their translation systems.  Machine translation clients require data confidentiality for:
> - Compliance with regulations and standards like HIPAA and GDPR
> - Compliance with security reviews
> - Competitiveness

> ### Data types
> - Training data - parallel data
> - Request data - original input

> ### Approaches
> - Client-specific - The data is not used to train models for any other clients.
> - No-trace - The request data is 
> - Auto-delete - The request data is cached, and deleted after a set time period, for example after 72 hours.
> - Delete on request - The training data or request data is deleted upon the client's request.
> - On-premise deployment ("on-prem")
> - On-device models

Comment (bittlingmayer) [reply business/data-confidentiality.md:7]: IMHO what's missing here a bit is some way to make it relative to the risks of TMS / CAT / human translators.

File: business/data-confidentiality.md [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+
+In systems with no confidentiality settings, input data can reappear in similar machine translation calls. Providers that offer no-trace never store data.
```
Comment (bittlingmayer) [inline business/data-confidentiality.md:9]: > input data can reappear in similar machine translation calls

This is an important point.  As usual, I tend towards structuring this, because this is a very different type of risk.

> ### Risk types
> Confidential data can be exposed when:
> - Training data is accessed.
> - Request data is accessed.
> - Training data is included as part of translation output.

File: business/products.md [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+and other features, as well as underlying architecture.
+
+
+The advantage of using a machine translation provider is that the service is already working and can be integrated immediately. The alternative is to train a model from scratch or to use a pre-trained model, and deploy and maintain a machine translation system.
+
+Unlike raw models, production machine translation systems usually provide language identification and sentence segmentation.
+
+Providers offer machine translation services via a platform, an API or both. Some providers offer limited services for free or paid full services.
+
+| Company | Product | Service |
```
Comment (bittlingmayer) [inline business/products.md:26]: I would try to make this more structured.

So make *Self-serve* and *API* and *Custom* columns, and then it's just a binary value.

But I would remove "Custom" and "API", because almost all of them offer that.

So the only thing that needs to be there is "Self-serve".  Therefore, it would also be possible to split them into 2 lists.

File: customization/parallel-data.md [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+Parallel data is necessary to train statistical and neural machine translation engines.
+
+### Challenges
+
+Parallel data is available in most widely spoken languages, but not available in other language pairs.
+
+Parallel data can have errors, like misaligned sentences, bad sentence segmentation, bad encodings, wrong or mixed language. Errors in parallel data are challenging because they affect the quality of the machine translation output. Parallel data errors can be solved via [filtering](customization/filtering.md).
+
+### Public parallel data repositories
+
```
Comment (bittlingmayer) [inline customization/parallel-data.md:28]: Europarl

File: customization/parallel-data.md [hunk 13]
```diff
@@ -0,0 +1,31 @@
...
+Parallel data is necessary to train statistical and neural machine translation engines.
+
+### Challenges
+
+Parallel data is available in most widely spoken languages, but not available in other language pairs.
+
+Parallel data can have errors, like misaligned sentences, bad sentence segmentation, bad encodings, wrong or mixed language. Errors in parallel data are challenging because they affect the quality of the machine translation output. Parallel data errors can be solved via [filtering](customization/filtering.md).
+
+### Public parallel data repositories
```
Comment (bittlingmayer) [inline customization/parallel-data.md:27]: Hmmm.  Maybe we split it into data repositories and data *sets*?

Comment (bittlingmayer) [reply customization/parallel-data.md:27]: Or make it a table with a column for which type it is

File: customization/parallel-data.md [hunk 14]
```diff
@@ -0,0 +1,31 @@
...
+
+Parallel data can be created by crawling and aligned monolingual test, and by [back-translation](customization/back-translation.md) or [back-copying](customization/back-translation.md).
+
+### Goals
+
+Parallel data is necessary to train statistical and neural machine translation engines.
+
+### Challenges
+
+Parallel data is available in most widely spoken languages, but not available in other language pairs.
```
Comment (bittlingmayer) [inline customization/parallel-data.md:23]: widely *written*, but even this is problematic, because there is not much parallel data for 2 big languages that don't have a lot of translation need.  (e.g. EN:PL >> HI:ES)

Also, parallel data is not *in* a language, it's *for* a language *pair*.

File: customization/parallel-data.md [hunk 15]
```diff
@@ -0,0 +1,31 @@
...
+| ----------- | ----------- |
+| File | Archivo |
+
+Parallel datasets can include translations for one or more language pairs, and be directioned or directionless.
+
+Parallel data can be created by crawling and aligned monolingual test, and by [back-translation](customization/back-translation.md) or [back-copying](customization/back-translation.md).
+
+### Goals
+
+Parallel data is necessary to train statistical and neural machine translation engines.
```
Comment (bittlingmayer) [inline customization/parallel-data.md:19]: is *used*

(There is back-translation, zero-shot etc etc...  Some innovations could happen.  So it's not strictly necessary, or may not be in the future.)

File: customization/parallel-data.md (hunk 12 above)
Comment (bittlingmayer) [reply customization/parallel-data.md:28]: OpenSubtitles

Comment (bittlingmayer) [reply customization/parallel-data.md:28]: JESC

Comment (bittlingmayer) [reply customization/parallel-data.md:28]: Hansard

Comment (bittlingmayer) [reply customization/parallel-data.md:28]: Mozilla

Comment (bittlingmayer) [reply customization/parallel-data.md:28]: CCMatrix
WikiMatrix

Comment (bittlingmayer) [reply customization/parallel-data.md:28]: WikiTitles

Comment (bittlingmayer) [reply customization/parallel-data.md:28]: VoxPopuli

Comment (bittlingmayer) [reply customization/parallel-data.md:28]: FLORES

File: business/products.md [hunk 16]
```diff
@@ -0,0 +1,47 @@
...
+- language support
+- accuracy
+- price
+- customization options
+- data confidentiality
+
+and other features, as well as underlying architecture.
+
+
+The advantage of using a machine translation provider is that the service is already working and can be integrated immediately. The alternative is to train a model from scratch or to use a pre-trained model, and deploy and maintain a machine translation system.
```
Comment (bittlingmayer) [inline business/products.md:20]: Let's change this to "An advantage" and "An alternative"...  To be more neutral and evergreen.

File: business/products.md [hunk 17]
```diff
@@ -0,0 +1,47 @@
...
+| Alibaba | Machine Translation | Character-based self-serve API |
+| Systran | Systran Translate Pro | Character-based self-serve API |
+| Unbabel | Unbabel | Volume- and content-based [hybrid translation](hybrid-translation.md) suscription |
+| Lilt | Lilt | User-based adaptive machine translation |
+| Tilde | TildeMT | Custom machine translation subscription |
+| Omniscien | Language Studio | Machine translation software platform |
+| KantanMT | Translate AI | Word-based custom machine translation |
+| Pangeanic | PangeaMT | Custom machine translation |
+| TextShuttle | TextShuttle AI Translation | Custom machine translation subscription |
+| MorphoLogic Globalization | Globalese | Custom machine translation subscription |
```
Comment (bittlingmayer) [inline business/products.md:46]: I believe the company is actually called Globalese
https://www.globalese-mt.com/impressum/

File: business/products.md [hunk 18]
```diff
@@ -0,0 +1,47 @@
...
+| Yandex | Translate API | Character-based self-serve API |
+| Microsoft | Translator Text API | Character-based self-serve API |
+| Amazon | Translate API | Character-based self-serve API |
+| ModernMT | API | Character-based self-serve API |
+| IBM | Watson Language Translator | Character-based self-serve API |
+| Baidu | 翻译API | Character-based self-serve API |
+| DeepL | API | Character-based self-serve API |
+| Alibaba | Machine Translation | Character-based self-serve API |
+| Systran | Systran Translate Pro | Character-based self-serve API |
+| Unbabel | Unbabel | Volume- and content-based [hybrid translation](hybrid-translation.md) suscription |
```
Comment (bittlingmayer) [inline business/products.md:39]: Seems it's called "Unbabel Translate"

https://www.google.com/search?q=%22unbabel+translate%22&client=firefox-b-d&sxsrf=AOaemvIx9VGVpAEgNMBqNUt40RLvhdi_yg%3A1631789609791&ei=KSJDYZfGL-n-7_UP_qWrwAI&oq=%22unbabel+translate%22&gs_lcp=Cgdnd3Mtd2l6EAMyBggAEAcQHjIGCAAQBxAeMgYIABAHEB4yBggAEAcQHjIGCAAQBxAeMgYIABAHEB4yBggAEAcQHjIICAAQCBAHEB46BwgAEEcQsANKBAhBGABQv0tY71FgslJoAnACeACAAVmIAd8EkgEBOJgBAKABAcgBA8ABAQ&sclient=gws-wiz&ved=0ahUKEwjX_qeAqoPzAhVp_7sIHf7SCigQ4dUDCA0&uact=5

File: business/products.md [hunk 19]
```diff
@@ -0,0 +1,47 @@
...
+| Lilt | Lilt | User-based adaptive machine translation |
```
Comment (bittlingmayer) [inline business/products.md:40]: ditto

https://www.google.com/search?client=firefox-b-d&q=%22lilt+translate%22

File: business/data-confidentiality.md [hunk 20]
```diff
@@ -4,12 +4,35 @@ description: Data confidentiality in machine translation
...
-**Data confidentiality** refers to how the machine translation providers use the specific data sent for translation. Providers can generate translations with or without storing the input. Machine translation providers need data to train and improve their translation service.
+**Data confidentiality** terms or policies govern how machine translation providers use client data.
 
-In systems with no confidentiality settings, input data can reappear in similar machine translation calls. Providers that offer no-trace never store data.
+Machine translation providers use client data to train and improve their translation systems. Machine translation clients require data confidentiality for:
 
-Free machine translation services have lower confidentiality levels than private machine translation services.
+- Compliance with regulations and standards like HIPAA and GDPR
+- Compliance with security reviews
+- Competitiveness
```
Comment (bittlingmayer) [inline business/data-confidentiality.md:13]: I guess "Competitiveness" can be removed, because that's why security guidelines exist.

But I meant, there are 3 distinct issues:
- consumer user data
- their B2B client data
- their own business data (e.g. new product launches)

File: business/products.md [hunk 21]
```diff
@@ -17,31 +17,46 @@ They vary by:
 and other features, as well as underlying architecture.
 
 
-The advantage of using a machine translation provider is that the service is already working and can be integrated immediately. The alternative is to train a model from scratch or to use a pre-trained model, and deploy and maintain a machine translation system.
+The alternative of using a machine translation provider is that the service is already working and can be integrated immediately. The alternative is to train a model from scratch or to use a pre-trained model, and deploy and maintain a machine translation system.
```
Comment (bittlingmayer) [inline business/products.md:20]: seems like a mistake to change this to "alternative"

Better "An advantage of" and "An alternative to"

**Review by bittlingmayer — APPROVED** [review]
